### DISCUSSION TITLE
Compare folder structure, ignore files?

### DISCUSSION THREAD

**Discussion by Pierre-de-la-Verre** [WinMerge/winmerge#629] [Q&A] [answered]
I just want to compare the structures of folders and want to ignore which files are in the folders and if the files are different. Is this possible?

**Comment by sdottaka** [comment, marked as answer]
Try specifying a filter that does not match any file, such as "* .aaaa", in the Select File or Folder window.

![image](https://user-images.githubusercontent.com/98126/107850928-746e8480-6e49-11eb-8305-df666e9b24bf.png)

**Comment by Pierre-de-la-Verre** [comment]
Thanks, works fine!

**Comment by o0Julia0o** [comment]
How can i only compare folders without subfolders or files(i have marked this field and quick files search, too. Ignore size, etc., too). I don´t want to compare the files.

I have 800 folders to compare. Next to them are 600 files. But the comparison takes until yet more then 2 hours.

**Reply by sdottaka** [reply]
I don't understand exactly what it means to compare folders without comparing files, but I understand that the problem is slowness.

If you are not interested in whether the contents of the files match exactly, try the comparison method "Modified Date and Size" as follows.

![image](https://user-images.githubusercontent.com/98126/108065467-67e46900-70a1-11eb-86c9-a515d37bd0aa.png)

If you select this, the comparison should end faster because it only compares the last modified date and size of the files without comparing the contents of the files.

**Reply by Pierre-de-la-Verre** [reply]
Maybe the question is meant:
"I have folder A with 700 subfolders and folder B with 800 subfolders. 
Compare only A with B. 
but not the files in A and B 
and not the subfolders in deeper levels."

**Reply by o0Julia0o** [reply]
yes, that´s the question. Which folders are different (withouth subfolders and without files)

**Comment by Pierre-de-la-Verre** [comment]
I think it is (nearly) the same as in my question above
- select folder A and folder b
- define a filter which will find no results (like (*.aaaaaaaaaaaaaa)
- don't select "Include subfolders"

That should do it.

**Comment by o0Julia0o** [comment]
Thx, that´s what i tried. After some hours i stopped the search and found the diffrence (1 folder) with my own eyes.

**Comment by Pierre-de-la-Verre** [comment]
strange .. - twice:
I created a folder "Left", containing empty folder "Folder_001" ... to "Folder_700, then removed "Folder_626".
Made the same for folder "Right", but removed "folder_072"

I set this:
![grafik](https://user-images.githubusercontent.com/13277944/108272055-04763c00-7172-11eb-97e5-7d842ece8c2b.png)
and got this in 190 ms:
![grafik](https://user-images.githubusercontent.com/13277944/108272133-2079dd80-7172-11eb-9a1a-484d2de8cca1.png)

Strange 1:
The slow (frozen) speed on your PC. (But my folders are really completely empty). Do you have usual speed on other compareing commands?

Strange 2:
I tried to set a "View" to display only the 2 different lines. But I saw either all lines or none ..

**Reply by o0Julia0o** [reply]
My folders are fully packed with stuff - lot´s of gigabytes. Maybe that´s the problem.
